docs/README.md

```diff
@@ -0,0 +1,14 @@
+# 💊  ReVanced Manager
+
+The official guide for ReVanced Manager.
+
+For bug reports, suggestions, or feature requests, open an issue [here](https://github.com/revanced/revanced-manager/issues/new/choose). If you wish to discuss the Manager further, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in our [Discord server](https://revanced.app/discord).
+
+## 📖 Table of contents
+
+1. [💼 Prerequisites](0_prerequisites.md)
+2. [🧩 Patching applications](1_patching-applications.md)
+3. [🧰 Managing patched applications](2_managing-patched-applications.md)
+4. [🔄 Updating ReVanced Manager](3_updating-revanced-manager.md)
+5. [🛟 Troubleshooting](4_troubleshooting.md)
+6. [🛠 Building from source](5_building-from-source.md)
```

Comment:
Nested lists could be helpful to differentiate groups of docs. Troubleshooting is separate of the guide for example.